common/scripted_triggers/th_scripted_triggers_for_danmaku_fight.txt

```diff
@@ -0,0 +1,778 @@
+th_has_danmaku_duels_unlocked = {
+	custom_trigger_tooltip = {
+		tooltip = th_has_danmaku_duels_unlocked_tt
+		OR = {
+			technology_group = tech_gensokyan_lunarian
+			technology_group = tech_lunarian
+			technology_group = tech_gensokyan
+		}
+	}
+}
+
+th_has_not_danmaku_duels_unlocked = {
+	custom_trigger_tooltip = {
+		tooltip = th_has_not_danmaku_duels_unlocked_tt
+		NOT = { technology_group = tech_gensokyan_lunarian }
+		NOT = { technology_group = tech_lunarian }
+		NOT = { technology_group = tech_gensokyan }
```

Comment:
Use `th_has_touhou_technology = no`

Reply:
Scripted triggers work not great with the no argument